### ISSUE TITLE
When running az functionapp config set --name "funcappname" --resource-group "rgname" --net-framework-version 8' receiving warning

### ISSUE THREAD

**Issue by SiddheshDesai06** [MicrosoftDocs/azure-devops-yaml-schema#308]
Refer my [SO answer here](https://stackoverflow.com/questions/78003954/azure-cli-invalid-net-runtime-version), When I am running 

```bash
az functionapp config set --name "funcappname" --resource-group "rgname" --net-framework-version 8
```

***I am receiving this warning:***

> WARNING: Invalid version: for runtime dotnet-isolated and os windows. Supported versions for runtime dotnet-isolated and os windows are: ['.4.8', '8', '7', '6']. Run 'az functionapp list-runtimes' for more details on supported runtimes.

***But the Function app is getting updated with the  `.NET 8 Isolated` but the warning is still getting recorded in the DevOps output console. This warning is not present when I run this code in my Local command prompt via Azure CLI commands:-***

DevOps output with warning but code is updating the setting successfully:-

![enter image description here](https://i.imgur.com/gvJjEfC.png)

**Local command output:-**

***Command:-***

```bash
az functionapp config set --name "funcappname" --resource-group "RG-Name" --net-framework-version 8
```

![enter image description here](https://i.imgur.com/F0J8Qk5.png)

**Comment by WilliamAntonRohm** [comment]
@SiddheshDesai06 -- Siddhesh, thank you for your detailed feedback.  Please consider this resource, if you haven't already:
- [Azure DevOps Services](https://azure.microsoft.com/support/devops/)
